frontend/public/components/container.tsx

```diff
@@ -379,40 +394,60 @@ const ContainerDetails: React.FC<ContainerDetailsProps> = (props) => {
 ContainerDetails.displayName = 'ContainerDetails';
 
 export const ContainersDetailsPage: React.FC<ContainerDetailsPageProps> = (props) => {
+  return (
+    <Firehose
+      resources={[
+        {
+          name: props.match.params.podName,
+          namespace: props.match.params.ns,
+          kind: 'Pod',
+          isList: false,
+          prop: 'obj',
+        } as FirehoseResource,
```

Comment:
Why is `as` needed here? We should generally avoid `as` because it often means there is a problem with the types somewhere.